### PR TITLE
Can we please add these two options

### PR DESCRIPTION
**Pull request by jzwar**
# Just some very small modifications to the show options

@j042 

```python
import numpy as np
import gustaf as gus

spline = gus.Bezier(
    degrees=[1,1],
    control_points=[
        [0, 0],
        [0, 1],
        [1, 0],
        [1, 1],
    ]
)
spline.elevate_degrees([0,1])
spline.cps[:] += np.random.rand(9,2) * 0.2 # some distortion

spline.show_options["c"] = "green"
spline.show_options["control_mesh_c"] = "darkgreen"
spline.show_options["control_points_c"] = "gray"
spline.show_options["control_mesh_lw"] = 1

```

### CODE REVIEW

**Comment by jzwar** [comment]
image

File: gustaf/spline/visualize.py [hunk 1]
```diff
@@ -247,10 +267,14 @@ def _vedo_showable(spline):
...
-        c_mesh.show_options["lw"] = 4
+        c_mesh.show_options["c"] = spline.show_options.get(
+            "control_mesh_c", "red"
+        )
+        c_mesh.show_options["lw"] = spline.show_options.get(
+            "control_mesh_lw", 4
+        )
         c_mesh.show_options["alpha"] = spline.show_options.get(
-            "control_points_alpha", 0.8
+            "control_points_alpha", 1.0
```
Comment (j042) [inline gustaf/spline/visualize.py:277]: how important is this?

Comment (jzwar) [reply gustaf/spline/visualize.py:277]: not, just thought it was a bug

Comment (j042) [reply gustaf/spline/visualize.py:277]: so, like 3?

Comment (j042) [reply gustaf/spline/visualize.py:277]: I think original intention was to make sure that spline body is prioritized

Comment (jzwar) [reply gustaf/spline/visualize.py:277]:
```suggestion
            "control_points_alpha", 0.8
```

**Review by j042 — APPROVED** [review]
nice fine tuning options

**Comment by j042** [comment]
> nice fine tuning options

Thank you! :)

**Pull request merged**